src/watchNPM.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request-promise')
+const _ = require('lodash')
+const ChangesStream = require('changes-stream')
+const clean = require('normalize-registry-metadata')
+const logger = require('~/src/logging').logger(module)
+
+// will hit up registry
+exports.checkWatcher = async function () { // TODO: fill the skeleton
+  console.log('todo')
+}
+
+// will publish to mq
+exports.handleChanges = async function ({channel, client, registry}) {
+  logger.info('saw change....todo handle logic')
+}
+
+exports.startNPMWatcher = async function ({channel, client, registry}) {
+  console.log(registry)
+  client.get('changes-stream', function (err, reply) {
```

Comment:
the original changes repo used redis to cache registry states (and the keys dynamically changed with time the registry change happened (or request made forget which off top of head) ), I had this in here at the moment since I may end up trying to do something like that when handling a change, not sure yet though. May throw it out. Still trying to figure out the flow, and truly understand what this service is supposed to do specifically. Trying to get some output out of changesStream to better understand everything. This PR is not inteneded to be merged in this state